src/FakerDotNet/Fakers/RickAndMortyFaker.cs

```diff
@@ -0,0 +1,41 @@
+﻿using FakerDotNet.Data;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace FakerDotNet.Fakers
+{
+    public interface IRickAndMortyFaker
+    {
+        string Character();
+        string Location();
+        string Quote();
+    }
+
+    internal class RickAndMortyFaker : IRickAndMortyFaker
+    {
+        public static readonly RickAndMortyData Data = new RickAndMortyData();
```

Comment:
You can make this `private`, as nothing outside this class will need this

Reply:
Sorry about that! Made it private now